### PR TITLE
Initial content from Alysha

### PR DESCRIPTION
**Pull request by Slimleashma**

### CODE REVIEW

File: index.rst
Comment (patrickingraham) [inline index.rst]: with different levels of recovery...?

File: index.rst
Comment (patrickingraham) [inline index.rst]: Are we still using the E-stop?

File: index.rst
Comment (patrickingraham) [inline index.rst:27]: Ok.. so this probably answers my question but I don't quite understand...

File: index.rst
Comment (patrickingraham) [inline index.rst]: I'm worried that someone might include the cryotiger (poly cold) as "instrument cooling electronics" .. maybe rephrase to "powering up the instruments required to support the instrument cooldown"

File: index.rst
Comment (patrickingraham) [inline index.rst]: The electronics to be powered on at this point in time consist of:

File: index.rst
Comment (patrickingraham) [reply index.rst]: In fact, you might want to add some sort of `..important::` note that says the instrument can be damaged by prematurely powering on the polycold system and extreme caution must be exercised to ensure everything is ready before starting to cool.

File: index.rst
Comment (patrickingraham) [inline index.rst]: not sure it matters, but it's a picoammeter... meaning it measures pico-amps.

File: index.rst
Comment (patrickingraham) [inline index.rst]: Why do you say this isn't used?
This is the hcu for the ATCamera, no? It reports and publishes the sensor values etc.

File: index.rst
Comment (patrickingraham) [reply index.rst]: Also, these get powered on by the PDU or a switch? I think you should probably list the PDU information just incase they're powered off there (which they probably should be).

File: index.rst
Comment (patrickingraham) [inline index.rst]: the sensor (CCD), the spectrograph optics and all other parts are not under vacuum.

File: index.rst
Comment (patrickingraham) [inline index.rst]: Sounds like a bit of a sentence fragment; maybe just add this to the previous sentence?
"The turbo pump, vacuum hose, power supplies, and pressure sensor are stored together on the floor and first shelf of the storage container in a labeled cardboard storage box"

File: index.rst
Comment (patrickingraham) [inline index.rst]: Consider:
...(as operating the ion pump at higher pressures significantly decreases it's operational lifespan).

File: index.rst
Comment (patrickingraham) [inline index.rst]: I think you want to use a reference here, something like,
as shown in :ref:`turbopump-setup`

Comment (patrickingraham) [reply index.rst]: bah, github jacks up the formatting, the name has to be surrounded by these things `

File: index.rst
Comment (patrickingraham) [inline index.rst]: isopropyl

using gloves and new kimwipes?

File: index.rst
Comment (patrickingraham) [inline index.rst:108]: hmm, probably best to make sure the instrument valve is closed before starting to attach anything... but it better be closed or someone screwed up bad!

File: index.rst
Comment (patrickingraham) [inline index.rst:112]: turbopump output valve? I didn't think there was one?

Comment (patrickingraham) [reply index.rst:112]: There is a vent which has to be closed... which is a black thumbscrew where the pipe enters the red box... but that should never really be opened.

File: index.rst
Comment (patrickingraham) [inline index.rst]: ...BEFORE ever opening the dewar valve.

File: index.rst
Comment (patrickingraham) [inline index.rst]: Remove the "Please" -- just so it's clear that it's not a request...

File: index.rst
Comment (patrickingraham) [inline index.rst]: rocker switch

File: index.rst
Comment (patrickingraham) [inline index.rst]: I think this is ambiguous... "evacuated completely" means (to me) that it'll be at a really high vacuum, not at atmospheric pressure.
If you want the gory details, consider something like:
"If the dewar has been warmed up but not opened, the pressure will be approximately the partial pressure of water vapour (~20 Torr). In the case the dewar has been serviced or opened, it may be at atmospheric pressure or under a slight positive pressure if purged with dry nitrogen."
If not, then just replace "evacuated completely" with "warmed up to ambient temperature" or something like that.

**Review by patrickingraham — DISMISSED** [review]
Sorry, didn't quite finish this.
It's looking good. Just a bunch of minor stuff for the most part. I'll finish the review tomorrow.

File: index.rst
Comment (Slimleashma) [reply index.rst]: Changed language. Clarified. 👍

File: index.rst
Comment (Slimleashma) [reply index.rst]: Both, until I can publish the LOTO procedure for everyone to use. 👍

File: index.rst
Comment (Slimleashma) [reply index.rst:27]: It needs to be under LOTO. Procedure forthcoming. I do not have approval to publish. 👍

File: index.rst
Comment (Slimleashma) [reply index.rst]: 👍

File: index.rst
Comment (Slimleashma) [reply index.rst]: 👍

File: index.rst
Comment (Slimleashma) [reply index.rst]: 👍

File: index.rst
Comment (Slimleashma) [reply index.rst]: 👍

File: index.rst
Comment (Slimleashma) [reply index.rst]: 👍

File: index.rst
Comment (Slimleashma) [reply index.rst]: 👍

File: index.rst
Comment (Slimleashma) [reply index.rst]: 👍

File: index.rst
Comment (Slimleashma) [reply index.rst]: 👍

File: index.rst
Comment (Slimleashma) [reply index.rst]: 👍

File: index.rst
Comment (Slimleashma) [reply index.rst:108]: 👍

File: index.rst
Comment (Slimleashma) [reply index.rst:112]: That's what it is called. 👍

File: index.rst
Comment (Slimleashma) [reply index.rst]: 👍

File: index.rst
Comment (Slimleashma) [reply index.rst:112]: That's what it is called. 👍

File: index.rst
Comment (Slimleashma) [reply index.rst]: 👍

File: index.rst
Comment (Slimleashma) [reply index.rst]: 👍

File: index.rst
Comment (Slimleashma) [reply index.rst]: 👍

**Comment by craiglagegit** [comment]
Looks good to me.  I am approving so it is ready to merge.

**Review by craiglagegit — APPROVED** [review]

**Pull request merged**
